runtime/kusama/src/weights/pallet_bags_list.rs

```diff
@@ -62,4 +62,7 @@ impl<T: frame_system::Config> pallet_bags_list::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
+	fn put_in_front_of() -> Weight {
+		0
+	}
```

Comment:
so I guess we will upgrade in a follow up or before runtime upgrade, but we should be careful that we actually upgrade it.
Because it happens that system isn't updated by our release process (https://github.com/paritytech/polkadot/issues/4290) , so we should keep a careful eye on it.

Reply:
Yup planning on doing a follow up immediatley after this was merged